### ISSUE TITLE
[BUG] "#define Y2_DRIVER_TYPE xxxx" but "Y2_SPEP_PIN" does not work in 2.1.2.2

### ISSUE THREAD

**Issue by SeanW2024** [MarlinFirmware/Marlin#26801]
### Did you test the latest `bugfix-2.1.x` code?

Yes, and the problem still exists.

### Bug Description

V2.1.2&V2.1.2.1 works well.In version 2.1.2.2, the corresponding pins did not generate pulse signals.In fact, if E1_STEP_PIN is defined in pin. h, when # define X2-DRIVER-TYPE A4988, compilation will prompt "Note: Auto assigned X2 STEP/DIR/ENABLE-PINs to unused En_STEP/DIR/ENABLE-PINs." However, when # define Y2-DRIVER-TYPE A4988,the compiler will not provide any prompts.

 

### Version of Marlin Firmware

2.1.2.2

### Bed Leveling

None

### Your Slicer

None

### Host Software

None

### Don't forget to include

- [X] A ZIP file containing your `Configuration.h` and `Configuration_adv.h`.

**Comment by ellensp** [comment]
In bugfix HAS_DUAL_Y_STEPPERS was removed from the code

but 2.1.2.2 still uses it, but setting this define was omitted from the 2.1.2.2 code

open up Marlin/src/inc/Conditionals_LCD.h

Find 
```
#ifdef Y2_DRIVER_TYPE
  #define HAS_Y2_STEPPER 1
#endif
```
replace it with
```
#ifdef Y2_DRIVER_TYPE
  #define HAS_Y2_STEPPER 1
  #define HAS_DUAL_Y_STEPPERS 1
#endif
```

This brings back the warning and from scanning the code should make it work again

**Comment by SeanW2024** [comment]
Thank you. Now,it works properly
________________________________
发件人: ellensp ***@***.***>
发送时间: 2024年2月18日 22:28
收件人: MarlinFirmware/Marlin ***@***.***>
抄送: SeanW2024 ***@***.***>; Author ***@***.***>
主题: Re: [MarlinFirmware/Marlin] "#define Y2_DRIVER_TYPE xxxx" but "Y2_SPEP_PIN" not work (Issue #26801)


In bugfix HAS_DUAL_Y_STEPPERS was removed

but 2.1.2.2 still uses it, but this was omitted from thr 2.1.2.2 code

open up Marlin/src/inc/Conditionals_LCD.h

Find

#ifdef Y2_DRIVER_TYPE
  #define HAS_Y2_STEPPER 1
#endif


replace it with

#ifdef Y2_DRIVER_TYPE
  #define HAS_Y2_STEPPER 1
  #define HAS_DUAL_Y_STEPPERS 1
#endif



―

**Comment by thisiskeithb** [comment]
Closing since this is already fixed in `bugfix-2.1.x`.

**Comment by thisiskeithb** [comment]
Patch for `2.1.x` submitted:
- https://github.com/MarlinFirmware/Marlin/pull/27024
